mirrormaker/README.md

```diff
@@ -32,4 +32,11 @@ The tool uses a Kafka consumer to consume messages from the source cluster, and
 
 `--timings.producer.config` - property file to configure embedded timings producer.  
 
-`--schema.registry.url` - Avro schema registry URL for requesting avro schemas. 
\ No newline at end of file
+`--schema.registry.url` - Avro schema registry URL for requesting avro schemas.
+ 
+ **Docker usage:**
+ 
+ 1. Install Docker [https://docs.docker.com/installation/#installation](https://docs.docker.com/installation/#installation)
+ 2. `cd $GOPATH/src/github.com/stealthly/go_kafka_client`
+ 3. Build docker image: `docker build -t stealthly/go_kafka_mirrormaker --file Dockerfile.mirrormaker`
```

Comment:
Seems like it should be "docker build -t stealthly/go_kafka_mirrormaker --file Dockerfile.mirrormaker ." (with a dot in the end). Otherwise looks good to me